docs/howtos/gatekeeper-migration.md

```diff
@@ -0,0 +1,555 @@
+---
+sidebar_label: Migrating Gatekeeper Policies to Kubewarden
+title: "Migrating Gatekeeper Policies to Kubewarden"
+description: "Migrating Gatekeeper Policies to Kubewarden"
+keywords: [kubewarden, gatekeeper]
+doc-persona:
+  [kubewarden-operator, kubewarden-integrator, kubewarden-policy-developer]
+doc-type: [howto]
+doc-topic: [operator-manual]
+---
+
+This guide shows you how to convert an existing Gatekeeper policy into a
+Kubewarden policy. This process involves two main steps:
+
+1. Compile the Rego program into a WebAssembly (Wasm) module.
+2. Distribute the WebAssembly module as a Kubewarden policy.
+
+The [Rego policies
+tutorial](docs/tutorials/writing-policies/rego/01-intro-rego.md) covers most of
+the build process for compiling Rego code into a WebAssembly module. This guide
+focuses on the step-by-step process of extracting Gatekeeper Custom Resource
+Definitions (CRDs) and migrating them into a functional Kubewarden policy. It
+uses a basic Gatekeeper demo policy Prerequisites:
+
+- [opa](https://github.com/open-policy-agent/opa/releases): you use this tool
+  to build the code into wasm. This guide was written using the `v1.5.1` version.
```

Comment:
It's good we're explicit about the version we're using. We can then ignore all the cases when someone uses `opa` < 1.0.0 later on.